Dockerfile

```diff
@@ -25,7 +25,7 @@ COPY src/*.py /ExampleModule/
 RUN /ExampleModule/ExampleModule.py
 # -----------------------------------
 
-# docker build --rm https://github.com/genepattern/ExampleModule.git#develop -f Dockerfile -t genepattern/example-module:2
+# docker build --rm https://github.com/genepattern/ExampleModule.git#develop -f Dockerfile -t genepattern/example-module:<tag>
```

Comment:
Do you use this <tag> somewhere else or is that just a way to denote the tag in this comment? If we have some kind of "version" variable/tag somewhere (I think maybe Alex does something like this in his scripts) we could even automate the version number used here (assuming the Docker container has the same tag as the module, which may be a wrong assumption), in the manifest, throughout the documentation -- we could read it from the *.version files 🤔  --> I know that I sometimes forget to replace that version number everywhere (and since some of those, like in the documentation, we put a URL that will be created upon release, it's hard to then go back and edit it).

Nothing to change here, just wanted to leave this thought as we are thinking about automating some of these processes.

Reply:
@bahill  -- Not sure if leaving a comment notifies you, but this should.

Reply:
I use that when I'm iterating over image builds. I set it to <tag> so that you don't accidentally overwrite a versioned tag

Reply:
As for automating... I think we probably want to leave docker image tags up to the creators - folks prefer this with GP modules too, we just haven't pulled out that mechanism from Jenkins yet. Ted did update the integrator to allow arbitrary versions though.